packages/admin/src/common/toolbar/titleitem/ToolbarTitleItem.tsx

```diff
@@ -0,0 +1,31 @@
+import { Typography } from "@material-ui/core";
+import * as React from "react";
+
+import { useStackApi } from "../../../stack";
+import { ToolbarItem } from "../item/ToolbarItem";
+
+interface ToolbarTitleItemProps {
+    title?: React.ReactNode;
+}
+const ToolbarTitleItem: React.FunctionComponent<ToolbarTitleItemProps> = ({ title }) => {
+    const stackApi = useStackApi();
+
+    return (
+        <>
+            {title ? (
+                <ToolbarItem>{title}</ToolbarItem>
```

Comment:
IMO this line is all we need.
```              
  <ToolbarItem>{title}</ToolbarItem>
```

Is extracting a title out of `stackApi` what the Toolbar should be able to do?

Reply:
Maybe there could be a helper for this functionality, e.g. `<ToolbarItemStackTitle />`, but I'm not sure if this should be the default functionality

Reply:
rendering things now with children